doc/.gitignore

```diff
@@ -1,3 +1,2 @@
 _build
 *.html
-*.adoc
```

Comment:
The point is that we *don't* want to accidentally commit the auto-generated adoc files.

Perhaps by adding a `doc/raddb/.gitignore` file, with appropriate content.

Reply:
@alandekok It's ready into the commit. I tested and didn't add any auto-generated *.adoc files.

![image](https://user-images.githubusercontent.com/254448/62327655-92eb7580-b487-11e9-993c-b567271a37de.png)